addons/test_discuss_full/__manifest__.py

```diff
@@ -12,7 +12,10 @@
         'calendar',
         'crm',
         'crm_livechat',
+        'hr_attendance',
+        'hr_fleet',
         'hr_holidays',
+        'hr_homeworking',
```

Comment:
Note: those were missing which made the query count incorrect locally/runbot depending on installed modules.
The foal of "_full" tests is to cover these to have deterministic values.